conf/category-list.json

```diff
@@ -8,6 +8,13 @@
         { "name": "Stylistic Issues", "description": "These rules relate to style guidelines, and are therefore quite subjective:" },
         { "name": "ECMAScript 6", "description": "These rules relate to ES6, also known as ES2015:" }
     ],
+    "deprecated": {
+        "name": "Deprecated",
+        "description": "These rules have been deprecated and replaced by newer rules.",
```

Comment:
For the "removed" rules we end the description with a colon instead of a dot. Should we do the same here for consistency?